### PR TITLE
Whitesource

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: project/Whitesource.scala [hunk 1]
```diff
@@ -13,14 +13,13 @@ object Whitesource extends AutoPlugin {
...
-   )
-
-   def majorMinor(version: String): Option[String] =
-     """\d+\.\d+""".r.findFirstIn(version)
+      val projectName = (moduleName in LocalRootProject).value.replace("-root", "")
+      projectName + "-" + (
+        if (isSnapshot.value)
+          if (gitCurrentBranch.value == "master") "master"
+          else "adhoc"
+        else "stable"
```
Comment (2m) [inline project/Whitesource.scala]: It is a bit unfortunate this this logic needs to be in every project that uses sbt-whitesource. Would it be possible to move that to the sbt plugin itself?

Comment (raboof) [inline project/Whitesource.scala]: I agree it's unfortunate, though looking at the current git branch might not be something we'd like to promote universally...

**Comment by patriknw** [comment]
@raboof ready to be merged?

**Comment by raboof** [comment]
Jup!

**Pull request merged**